enrolments/models.py

```diff
@@ -88,7 +88,10 @@ class Enrolment(models.Model):
         on_delete=models.PROTECT,
     )
     enrolled_class = models.ForeignKey(
-        "enrolments.Class", on_delete=models.PROTECT, related_name="enrolments"
+        "enrolments.Class",
+        on_delete=models.PROTECT,
+        related_name="enrolments",
+        null=True,
```

Comment:
if we're allowing this to be null, [this validator](https://github.com/uwblueprint/project-read-backend/blob/2501a5d4e70adc1508714a009128b3351953a13c/enrolments/validators.py#L29) may raise an error - can we add a null check to that?

Reply:
Ooh did not know about that validator very good call! Fixed 😁